ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
@douglashall : I'm pretty sure a majority of the unit tests in ab_testing are hitting the test database.
@rascalking : I don't think we'll want to grant createdb privileges to roles that will be accessing any Django project db.  I believe, as doug alluded to earlier, that we'd have to do so for Django to use postgres as the test db engine.  I think that maybe rules out running unit tests against postgres, even if we're developing locally against a postgres instance.  Maybe we should continue the discussion offline?